pkg/utils/config.go

```diff
@@ -300,11 +300,7 @@ func LoadConfigFile(config string) (loaded bool, err error) {
 	}
 	log.Debugf("Will use config from %s", config)
 	if _, err = os.Stat(config); os.IsNotExist(err) {
-		if err = GenerateConfigFile(config); err != nil {
-			return false, fmt.Errorf("fail in generate yaml: %s", err.Error())
-		} else {
-			log.Infof("Config file generated: %s", config)
-		}
+		log.Fatal("no config, please run 'eden config add default'")
```

Comment:
please run 'eden config add' command or switch to existing config

Reply:
'eden config add' isn`t valid command, you must provide name (eden config add <name> [flags]).
Also, "switch to existing config", not valid for this place, because any existing config is just a diff onto default config. So, we need default config and phrase "no config, please run 'eden config add default'" is the valid variant.